dev/appveyor-install-dependencies.ps1

```diff
@@ -115,7 +115,7 @@ $env:Path += ";$env:HADOOP_HOME\bin"
 Pop-Location
 
 # ========================== R
-$rVer = "3.4.1"
+$rVer = "3.5.1"
 $rToolsVer = "3.4.0"
```

Comment:
is there update to rtool?

Reply:
Oh~ latest looks 3.5. Let me try to update aswell shortly.